libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -830,8 +856,13 @@ export function Workspace () {
             </div>
           </div>
         }
+        {showContextMenu && <FileExplorerRootMenu focusContext={focusContext} 
```

Comment:
Second context menu is not needed.

Reply:
Okay, so how should I show the context menu?

Reply:
See gitter comments.